.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Prettier formatting errors
+32743cbd2f5dead77fd6ef7be25105f33c370f29
```

Comment:
is there a command that can force-format everything? we did it in IPA code too, the PR was large, but one-time action to clean up the entire code base was worth it